components/ReserveForm/index.js

```diff
@@ -27,24 +28,37 @@ export default function ReserveForm({
         name="number_of_guests"
         id="number_of_guests"
         min="1"
-        max={remainingSeats}
+        max={availableSeats}
+        defaultValue={defaultData?.number_of_guests}
         required
       ></input>
       <label htmlFor="name">Name:</label>
-      <input type="text" name="name" id="name" required></input>
+      <input
+        type="text"
+        name="name"
+        id="name"
+        defaultValue={defaultData?.name}
+        required
+      ></input>
       <label htmlFor="email">Email Adresse:</label>
-      <input type="email" name="email" id="email"></input>
+      <input
+        type="email"
+        name="email"
+        id="email"
+        defaultValue={defaultData?.email}
+      ></input>
       <label htmlFor="phone">Telefonnummber:</label>
       <input
         maxLength="11"
         minLength="6"
         type="tel"
         name="phone"
         id="phone"
+        defaultValue={defaultData?.phone}
         required
       ></input>
-      <StyledButton type="submit" aria-label="Knopf um zu reservieren">
-        Reservieren
+      <StyledButton type="submit" aria-label="Knopf um zu einreichen">
+        Einreichen
```

Comment:
I'd rather say: Abschicken or Speichern. Einreichen is very formal...when you apply for something at the Amt for example.